packages/test/test-end-to-end-tests/src/test/gc/gcReferenceUpdatesInLocalSummary.spec.ts

```diff
@@ -79,6 +80,7 @@ describeFullCompat("GC reference updates in local summary", (getTestObjectProvid
 
     const runtimeOptions: IContainerRuntimeOptions = {
         summaryOptions: {
+            disableSummaries: true,
```

Comment:
> disableSummaries: true, [](http://example.com/codeflow?start=11&length=24)

Do we need it ? I think the state:disable does it for us.

Reply:
It's for back-compat. Older container runtime don't have state: "disabled" yet.

Reply:
I see  :)